plugins/module_utils/network/sonic/config/lldp_interfaces/lldp_interfaces.py

```diff
@@ -80,6 +82,20 @@ def get_lldp_interfaces_facts(self):
             return []
         return lldp_interfaces_facts
 
+    def convert_vlan_list(self, vlan_lst):
+        """
+        Convert a list of VLANs to a new list.
+        :rtype: vlan_lst (list): A list of VLANs.
+        :returns:A new list containing the converted VLANs.
+        """
+        result = []
+        for vlan in vlan_lst:
+            try:
+                result.append(int(vlan))
+            except ValueError:
+                result.append(vlan)
```

Comment:
Exceptions cause expenditure of many CPU cycles and should not be used instead of a conditional branch structure. Please use something like the following instead:

```suggestion

            if ".." in vlan:
                result.append(vlan)
            else:   
                result.append(int(vlan))
```